tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_ops.py

```diff
@@ -752,7 +752,7 @@ def embedding_lookup_unique(params,
 
 
 def embedding_lookup_sparse(
-    params,
+    embedding_weights,
```

Comment:
We need not change the parameters of public API, but we must also keep consistency with TensorFlow's same-name API.